### ISSUE TITLE
ReferenceUsedNamesOnly with allowFallbackGlobalConstants=false reports invalid errors for multiple declares

### ISSUE THREAD

**Issue by janedbal** [slevomat/coding-standard#239]
Following code
```php
<?php declare(strict_types = 1, ticks = 1, encoding = 'UTF-8');
```
with following config
```xml
<rule ref="SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly">
	<properties>
		<property name="allowFallbackGlobalConstants" type="bool" value="false"/>
	</properties>
</rule>
```
results in errors: 

```
Constant ticks should not be referenced via a fallback global name, but via a use statement.
Constant encoding should not be referenced via a fallback global name, but via a use statement
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/08dd8076876e33524d642529d4adae985b3bf3fb.
